### ISSUE TITLE
Speed hack frame drop Cheat engine version 7.5

### ISSUE THREAD

**Issue by Astriixs** [cheat-engine/cheat-engine#2766]
I'm trying to enable the speed hack in a game. (The game being Satisfactory)

When I click the check box to enable speed hack my game goes from a stable 120 fps on max graphics to a not-so-stable 20 - 50 fps while on 1 speed. This gets even worse when I turn up the speed or lower it decreasing to 1 - 10 fps (V-Sync off Btw).

After further investigation, I have concluded that the frame drops only seem to occur Satisfactory. I have tested the speed hacks in many games including The Forest Sons of the Forest (no frame drop) Raft (no frame drop) Grounded (no frame drop) Ark Survival Evolved (no frame drop). all the games stated were running on max graphics and tested on different speeds yet no frame drops but when I run Satisfactory on low graphics there is a massive frame drop.

Basically what I want to know is why does it only frame drop only in satisfactory and how can I fix it.

(Satisfactory in on unreal engine 5 if that means anything to anyone).

**Comment by githubsearcher11** [comment]
hey, same here. i tried with following games: cyberjunk(REDengine), UE5 based games, Unity mono based games (all of these are SP games, not online)
most of games starts to make stuttering, sudden frame drops when you enables speedhacking. worst part is even if you disables speedhacking this symptom continues. i suspect it doesn't unhooks speedhacking library on disabling.

this does not happens with ancient version of CE, so i think recent modification of speedhack method makes this issue.

**Comment by pilot2254** [comment]
just switch to older CE version

**Comment by Kyogre** [comment]
> this does not happens with ancient version of CE

please tell, what version?
